etk/unit_tests/test_extractions_using_spacy.py

```diff
@@ -68,18 +67,33 @@ def test_extraction_from_age_spacy(self):
 
             self.assertEquals(sorted(extracted_ages), sorted(t['correct']))
 
-    def test_extraction_from_social_media(self):
-        for t in self.ground_truth['social_media']:
+    # def test_extraction_from_social_media(self):
+    #     for t in self.ground_truth['social_media']:
 
-            crf_tokens = self.c.extract_tokens_from_crf(
-                self.c.extract_crftokens(t['content']))
+    #         crf_tokens = self.c.extract_tokens_from_crf(
+    #             self.c.extract_crftokens(t['content']))
+
+    #         nlp_doc = self.c.nlp(crf_tokens)
+
+    #         extracted_social_media_handles = spacy_social_media_extractor.extract(
+    #             nlp_doc, self.c.matchers['social_media'])
+
+    #         # print extracted_social_media_handles
+
+    # def test_extraction_from_address_spacy(self):
+    #     for t in self.ground_truth['address']:
+
+    #         crf_tokens = self.c.extract_tokens_from_crf(
+    #             self.c.extract_crftokens(t['content']))
+    #         nlp_doc = self.c.nlp(crf_tokens)
 
-            nlp_doc = self.c.nlp(crf_tokens)
+    #         extracted_addresses = spacy_address_extractor.extract(
+    #             nlp_doc, self.c.matchers['address'])
 
-            extracted_social_media_handles = spacy_social_media_extractor.extract(
-                nlp_doc, self.c.matchers['social_media'])
+    #         extracted_addresses = [address['value']
+    #                                for address in extracted_addresses]
 
-            #print extracted_social_media_handles
+    #         # print extracted_addresses
 
 if __name__ == '__main__':
     unittest.main()
```

Comment:
Take a look at test cases using dictionary for example, add unit tests to run spacy by calling c.process(doc) as well. 
Use the actual extraction_config